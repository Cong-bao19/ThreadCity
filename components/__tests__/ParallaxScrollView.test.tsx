import React from 'react';
import { create } from 'react-test-renderer';
import ParallaxScrollView from '../ParallaxScrollView';
import { useColorScheme } from '@/hooks/useColorScheme';

// Mock `useColorScheme` hook
jest.mock('@/hooks/useColorScheme', () => ({
  useColorScheme: jest.fn(),
}));

describe('ParallaxScrollView', () => {
  it('renders correctly', () => {
    // Mock trả về giá trị của `useColorScheme`
    (useColorScheme as jest.Mock).mockReturnValue('light');

    const tree = create(
      <ParallaxScrollView headerImage={<></>} headerBackgroundColor={{ light: '#fff', dark: '#000' }}>
        <></>
      </ParallaxScrollView>
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});
