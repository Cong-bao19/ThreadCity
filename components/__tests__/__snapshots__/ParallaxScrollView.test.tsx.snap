// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParallaxScrollView renders correctly 1`] = `null`;
